pkg/join/options.go

```diff
@@ -44,6 +44,9 @@ type Options struct {
 	CableDriver                   string
 	CoreDNSCustomConfigMap        string
 	BrokerURL                     string
+	HTTPProxy                     string
+	HTTPSProxy                    string
+	HTTPNoProxy                   string
```

Comment:
How about

```suggestion
	NoProxy                   string
```

to match `httpproxy.Config`?